media/system/js/share-uncompressed.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @package         Joomla.JavaScript
+ * @copyright       Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+/**
+ * Calls the sending process of the config class
+ */
+jQuery(document).ready(function ($)
+{
+    $('#sharebutton').click(function ()
+    {
+        var share_draft = {
+            articleId  : $().val(),
+            title: $('jform_title').val(),
+            sharetoken  : $().val(),
```

Comment:
@nikitadhiman Do we need the sharetoken here? Since you already have the article ID you can retrieve the token from the database. So I think this can be left out and the Article ID and title should be enough.